Dev/kduran/us vbs22 41 deshabilitación y eliminación de usuario

kendallduranv opened the pull request:
Added the functionality to delete (logical delete) of users and list users. Also made the unit test for both of these tasks

kendalljdv committed:
Listar y borrar usuarios- List of users and delete users

Added the logic to delete user (logical delete) and the test to list users and delete an user. Also fixed some anotations of the brach thatr i use to copy this one

kendalljdv committed:
Listar usuarios y eliminar usarios

Added some logic to  deletes users and list users, country Rest controller  added

kendalljdv committed:
Update application.properties

kendalljdv committed:
Merge branch 'development' into dev/kduran/US-VBS22-41-Deshabilitación_y_eliminación-de-usuario

kendalljdv committed:
Merge branch 'development' into dev/kduran/US-VBS22-41-Deshabilitación_y_eliminación-de-usuario

kendalljdv committed:
Fixes of the merge and auth sign up fixed

kendalljdv committed:
Delete comment

kendallduranv committed:
Merge branch 'development' into dev/kduran/US-VBS22-41-Deshabilitación_y_eliminación-de-usuario

kendallduranv commented:
@nefi-cenfo  @mur-tz @DieS06 @ichivancrCenfotec  please could  you check this branch, if it's ok to merge it to the development

nefi-cenfo commented on demo/src/main/java/com/project/demo/rest/country/CountryRestController.java:
```diff
@@ -0,0 +1,31 @@
+    @Autowired
+    private CountryRepository countryRepository;
+
+//    @GetMapping
```
Is this commented block needed?

kendallduranv replied:
now it is not needed, could be deleted

nefi-cenfo commented on demo/src/main/java/com/project/demo/rest/auth/AuthRestController.java:
```diff
@@ -62,14 +66,33 @@ public ResponseEntity<LoginResponse> authenticate(@RequestBody User user) {
+                country.setCountryCode("Default Country Code");
+                country.setCurrency(new Currency());
+                country.setOperational(true);
+                country = countryRepository.save(country);
```
Is it safe to have the repository call here since Juan changed the architecture?

kendallduranv replied:
could be deleted

kendalljdv committed:
Fixes

Fixed those files erasing the commented code and a unsafe call to the repository
